Description: gcc is complaining about no format (-Werror=format-security)
Bug: b-g-o: #661016
diff -uNr gnome-settings-daemon-3.0.3.orig/plugins/automount/gsd-autorun.c gnome-settings-daemon-3.0.3/plugins/automount/gsd-autorun.c
--- gnome-settings-daemon-3.0.3.orig/plugins/automount/gsd-autorun.c	2011-08-25 05:46:50.000000000 -0400
+++ gnome-settings-daemon-3.0.3/plugins/automount/gsd-autorun.c	2011-10-05 15:20:25.303752355 -0400
@@ -503,7 +503,7 @@
 							 GTK_BUTTONS_OK,
 							 "%s",
 							 primary);
-			gtk_message_dialog_format_secondary_markup (GTK_MESSAGE_DIALOG (dialog), error->message);
+			gtk_message_dialog_format_secondary_markup (GTK_MESSAGE_DIALOG (dialog), error->message, NULL);
 			
 			gtk_widget_show (GTK_WIDGET (dialog));
 			g_signal_connect (dialog, "response",
